RosettaCore/src/main/java/com/hubspot/rosetta/Rosetta.java

```diff
@@ -17,8 +17,8 @@
 public enum Rosetta {
   INSTANCE;
 
-  private final List<Module> modules = new CopyOnWriteArrayList<Module>(defaultModules());
-  private final AtomicReference<ObjectMapper> mapper = new AtomicReference<ObjectMapper>(cloneAndCustomize(new ObjectMapper()));
+  private static final List<Module> MODULES = new CopyOnWriteArrayList<Module>(defaultModules());
+  private static final AtomicReference<ObjectMapper> MAPPER = new AtomicReference<ObjectMapper>(cloneAndCustomize(new ObjectMapper()));
```

Comment:
Would it make more sense for the static methods in this class to reference `INSTANCE.modules`/`INSTANCE/.mapper` instead of making these static? This class is starting to toe the line between enum singleton and static util class IMHO